Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -825,8 +826,10 @@ public static void HandleWeapons(Session session, params string[] parameters)
             {
                 foreach (uint weenieId in weaponsTest)
                 {
-                    WorldObject loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
+                    AceObject lootAceObject = (AceObject)DatabaseManager.World.GetAceObjectByWeenie(weenieId).Clone(GuidManager.NewItemGuid().Full);
+                    WorldObject loot = new GenericObject(lootAceObject);
```

Comment:
It does part of it.  The DB layer should be assigning the objectGuid of the child objects, but it doesn't hurt (and is probably better) to do a deep reassignment of the guid.